README.md

```diff
@@ -1,9 +1,33 @@
 # cincinnati
 
-[![Join the chat at https://gitter.im/nodeschool/cincinnati](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/nodeschool/cincinnati?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Join the chat at https://gitter.im/nodeschool/cincinnati](https://badges.gitter.im/Join%20Chat.svg)][gitter]
 
 NodeSchool chapter for Cincinnati.
 
+
+## Getting Started
+
+Here is a list of helpful resources if you are new to node school events.
```

Comment:
Change "node school" to "NodeSchool" to match how it is throughout the document.